<launch>

	<arg name="port" default="/dev/ttyUSB0" />
	<arg name="baud" default="115200" />
	<arg name="channels" default="[left_motor, right_motor]" />

	<arg name="wheel_radius" default="0.15875" />
	<arg name="base_radius" default="0.35560" />

	<arg name="cmd_vel_topic" default="/cmd_vel" />

<!--  	<node pkg="roboteq_driver" type="driver_node" name="roboteq_driver" output="screen">
		<rosparam param="port">$(arg port)</rosparam>
		<rosparam param="baud">$(arg baud)</rosparam>
		<rosparam param="channels">$(arg channels)</rosparam>
	</node> -->

	<node pkg="igvc_vel_controller" type="vel_controller_node" name="vel_controller" output="screen">
		<rosparam param="channels" subst_value="True">$(arg channels)</rosparam>
		<rosparam param="wheel_radius" subst_value="True">$(arg wheel_radius)</rosparam>
		<rosparam param="base_radius" subst_value="True">$(arg base_radius)</rosparam>
		<rosparam param="cmd_vel_topic" subst_value="True">$(arg cmd_vel_topic)</rosparam>
	</node>
</launch>